app/qml/main.qml

```diff
@@ -164,9 +164,17 @@ ApplicationWindow {
       }
     }
 
+    function isGpsAccuracyLow() {
+      return (positionKit.accuracy <= 0)  || (positionKit.accuracy > __appSettings.gpsAccuracyTolerance)
+    }
+
     function getGpsIndicatorColor() {
         if (positionKit.accuracy <= 0) return InputStyle.softRed
-        return positionKit.accuracy < __appSettings.gpsAccuracyTolerance ? InputStyle.softGreen : InputStyle.softOrange
+        return isGpsAccuracyLow() ? InputStyle.softOrange : InputStyle.softGreen
+    }
+
+    function showGpsAccuracyWarning() {
+        return (__appSettings.gpsAccuracyWarning && (digitizing.useGpsPoint || !digitizing.manualRecording) && isGpsAccuracyLow() )
```

Comment:
It seems that this is not working. I switched off the `gpsAccuracyWarning` in settings and still can see the message (now it actually does not disappear at all)
![ezgif com-gif-maker (3)](https://user-images.githubusercontent.com/22449698/110331643-a9709000-801f-11eb-982f-3104c60fc5fe.gif)